src/ArduinoPackageManager.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as vscode from 'vscode';
+
+export interface BoardInfo {
+  additionalUrl: string;
+  packageName: string;
+  architecture: string;
+}
+const BOARD_INFO: {[key: string]: BoardInfo} = {
+  devkit: {
+    additionalUrl:
+        'https://raw.githubusercontent.com/VSChina/azureiotdevkit_tools/master/package_azureboard_index.json',
+    packageName: 'AZ3166',
+    architecture: 'stm32f4'
+  }
+};
+
+export class ArduinoPackageManager {
+  private static async setAdditionalUrl(url: string) {
+    const existedUrls = vscode.workspace.getConfiguration().get<string[]>(
+        'arduino.additionalUrls');

```

Comment:
use constant since it is used in several places

Reply:
We need design constant better. Currently constant is not in good structure. Let's make it in a standalone task.